web/src/containers/AssurancesAndCompliance.js

```diff
@@ -72,43 +68,48 @@ class AssurancesAndCompliance extends Component {
                 {apdSections[name].map(
                   ({ title, checked, explanation }, index) => (
                     <div key={title} className="mt2">
```

Comment:
`mt2` is from the old atomic system -- can we replace with ` ds-u-margin-top--2`? (There's also an `mb3` on line 66, but GH won't let me comment on it since it wasn't changed in this PR.)

Also, can we make this `div` a `fieldset`? ([`fieldset` is the only semantic parent for `legend`, weirdly enough](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/legend).)

Reply:
Oh, I guess.